src/Products/Comparison/Service/ComparisonServiceImpl.cs

```diff
@@ -175,10 +170,9 @@ public PageDescriptionEntity LoadDocumentPage(PostedDataEntity postedData)
                 loadedPage.width = resultImages[pageNumber - 1].Width;
 
             }
-            catch (System.Exception ex)
+            catch (Exception)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add logic to this catch clause or eliminate it and rethrow the exception automatically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5038425)